.devcontainer/.env

```
@@ -0,0 +1,2 @@
+# Ruby version
+VARIANT=3.1.3
```

```suggestion
VARIANT=3.1.4
```
For the dev container?